src/component/machine/Toyocut.jsx

```diff
@@ -1,11 +1,16 @@
-import React from "react";
-import { Button, Table } from "react-bootstrap";
-import { PrinterFill, TrashFill } from "react-bootstrap-icons";
+import React, { useState } from "react";
+import { PrinterFill, TrashFill, Truck } from "react-bootstrap-icons";
+import { Button } from "react-bootstrap";
 import { confirmationDialog } from "../../common/ConfirmationDialog";
 import axiosInstance from "../../axiosConfig";
 import { toast } from "react-toastify";
+import { AgGridReact } from "ag-grid-react";
+import "ag-grid-community/styles/ag-grid.css";
```

Comment:
no need to import ag-grid styles again for each page, already imported in main file gloabally